src/extra/gpkg/read.jl

```diff
@@ -0,0 +1,273 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+function gpkgread(fname; layer=1)
+  db = SQLite.DB(fname)
+  assertgpkg(db)
+  geom = gpkgmesh(db, ; layer)
+  attrs = gpkgmeshattrs(db, ; layer)
+  GeoTables.georef(attrs, geom)
+end
+
+function assertgpkg(db)
+  appid = DBInterface.execute(db, "PRAGMA application_id;") |> first |> only
+  userversion = DBInterface.execute(db, "PRAGMA user_version;") |> first |> only
```

Comment:
These are not used? Try to rename to `id` and `version` if they are to be used in future revisions.